wled00/audio_source.h

```diff
@@ -102,8 +102,8 @@ class I2SSource : public AudioSource {
             .mode = i2s_mode_t(I2S_MODE_MASTER | I2S_MODE_RX),
             .sample_rate = _sampleRate,
             .bits_per_sample = I2S_SAMPLE_RESOLUTION,
-            .channel_format = I2S_CHANNEL_FMT_ONLY_LEFT,
-            .communication_format = i2s_comm_format_t(I2S_COMM_FORMAT_I2S | I2S_COMM_FORMAT_I2S_MSB),
+            .channel_format = I2S_CHANNEL_FMT_ONLY_RIGHT,
+            .communication_format = I2S_COMM_FORMAT_I2S,
```

Comment:
breaking change, needs further discussions. 
@FHeilmann : maybe we need to offer two I2S drivers here, one for right channel, and one for left channel.